contracts/Inbox.sol

```diff
@@ -18,44 +18,16 @@ import {Semver} from "./libs/Semver.sol";
  * @dev Validates intent hash authenticity and executes calldata. Enables provers
  * to claim rewards on the source chain by checking the fulfilled mapping
  */
-contract Inbox is IInbox, Eco7683DestinationSettler, Ownable, Semver {
+contract Inbox is IInbox, Semver {
     using TypeCasts for address;
     using SafeERC20 for IERC20;
 
-    // Mapping of intent hash on the src chain to its fulfillment
-    mapping(bytes32 => ClaimantAndBatcherReward) public fulfilled;
-
-    // Mapping of solvers to if they are whitelisted
-    mapping(address => bool) public solverWhitelist;
-
-    // address of local hyperlane mailbox
-    address public mailbox;
+    bytes4 public constant IPROVER_INTERFACE_ID = 0xd8e1f34f; //type(IProver).interfaceId
 
-    // Is solving public
-    bool public isSolvingPublic;
+    // Mapping of intent hashes to their claimant address
+    mapping(bytes32 => address) public fulfilled;
 
-    // minimum reward to be included in a fulfillHyperBatched tx, to be paid out to the sender of the batch
-    uint96 public minBatcherReward;
-
```

Comment:
The constructor no longer initializes any access control mechanism. Consider retaining some form of administrator role for potential future upgrades or emergency measures.